.github/CODEOWNERS

```diff
@@ -4,3 +4,9 @@
 
 # The owners for all files in the repo.
 * @googleapis/cloud-cxx-owners
```

Comment:
Q.  Why shouldn't the `*` be a `/`?

Reply:
We should use `*` because the example linked in the PR description uses `*`.

But yeah, I agree with you that `/` should mean all files and subdirectories.

Reply:
https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/customizing-your-repository/about-code-owners#codeowners-syntax

https://git-scm.com/docs/gitignore#_pattern_format

The key section is (I think):

> If there is a separator at the beginning or middle (or both) of the pattern, then the pattern is relative to the directory level of the particular .gitignore file itself. Otherwise the pattern may also match at any level below the .gitignore level.

So, `*` does not have a `/` and therefore applies to any file in any directory and matches all files. Anything with a `/` would not match the files in subdirectories.

Maybe, I just read some docs, did not test the code or anything.